src/control/server/storage/bdev/backend.go

```diff
@@ -144,14 +145,17 @@ func createHugePageWalkFunc(hugePageDir string, stat statFn, remove removeFn, co
 
 		matches := re.FindStringSubmatch(info.Name())
 		if len(matches) != 2 {
+			log.Debugf("walk func: unexpected name, skipping %s", path)
 			return nil // skip files not matching expected pattern
 		}
 		// PID string will be the first submatch at index 1 of the match results.
 
 		if isActive, err := isPIDActive(matches[1], stat); err != nil || isActive {
+			log.Debugf("walk func: active owner proc, skipping %s", path)
```

Comment:
@tanabarr , would it be too tricky to also print owner PID, and may be also the owner command name ?